conduce/cli.py

```diff
@@ -316,6 +316,169 @@ def get_dataset_transactions(args):
+    del vars(args)['async_processing']
+
+    if all_backends:
+        backend_ids = api.list_dataset_backends(dataset_id, **request_kwargs(**vars(args)))
+
+    if async_processing or args.transaction or args.all:
+        for backend_id in backend_ids:
+            response = api.process_transactions(dataset_id, backend_id, **vars(args))
+            print('Processing... check status at {}'.format(response.headers['location']))
+    else:
+        min_tx = args.min
+        del vars(args)['min']
+        max_tx = args.max
+        del vars(args)['max']
+        del vars(args)['all']
+        del vars(args)['transaction']
+
+        max_transaction = max_tx or api.get_transactions(dataset_id, count=True, **vars(args))['count']
+        for backend_id in backend_ids:
+            min_transaction = min_tx or api.get_dataset_backend_metadata(dataset_id, backend_id, **vars(args))['transactions']
+            idx = 1
+            while min_transaction + idx <= max_transaction:
+                print("Processing transaction {} of {} on {}...".format(min_transaction + idx, max_transaction, backend_id))
+                response = api.process_transactions(dataset_id, backend_id, transaction=(min_transaction + idx), **vars(args))
+                api.wait_for_job(response.headers['location'])
+                idx += 1
+
+
+def enable_auto_processing(args):
+    dataset_id = args.dataset_id
+    del vars(args)['dataset_id']
+    backend_id = args.backend_id
+    del vars(args)['backend_id']
+    enable = not args.disable
+    del vars(args)['disable']
+
+    return api.enable_auto_processing(dataset_id, backend_id, enable=enable, **vars(args))
+
+
+def set_default_backend(args):
+    dataset_id = args.dataset_id
+    del vars(args)['dataset_id']
+    backend_id = args.backend_id
+    del vars(args)['backend_id']
+    return api.set_default_backend(dataset_id, backend_id, **vars(args))
+
+
+def remove_dataset_backends(args):
+    if len(args.backend_ids) == 0 and not args.all:
+        raise ValueError('You must provide a list of backend IDs or pass --all')
+
+    dataset_id = args.dataset_id
+    del vars(args)['dataset_id']
+    backend_ids = args.backend_ids
+    del vars(args)['backend_ids']
+    remove_all = args.all
+    del vars(args)['all']
+    force = args.force
+    del vars(args)['force']
+
+    if remove_all:
+        backend_ids = api.list_dataset_backends(dataset_id, **vars(args))
+        if not force:
+            answer = input("Removing {} backends! Are you sure (Y/n): ".format(len(backend_ids)))
+            if answer is not 'Y':
+                return
+
+    for backend_id in backend_ids:
+        location = api.remove_dataset_backend(dataset_id, backend_id, **vars(args)).headers['location']
+        print('Removing {}: check status at {}'.format(backend_id, location))
+
+
+def list_dataset_backends(args):
+    dataset_id = args.dataset_id
+    del vars(args)['dataset_id']
+    backends = args.backend_ids
```

Comment:
It seems weird for a method that is supposed to list backends to optionally take a list of backends.

I get what you're going for, but it feels like that should be a separate method.

Reply:
yeah, and i suppose it only works with the `-v` option

Reply:
I'll add a story to address this

Reply:
SKW-4227